components/delegations/DelegateCard.tsx

```diff
@@ -77,76 +79,74 @@ export default function DelegateCard({ delegate }: PropTypes): React.ReactElemen
 
             {!showLinkToDetail && (
               <Box>
-                <DelegateLastVoted delegate={delegate} />
+                {/* <DelegateLastVoted delegate={delegate} /> */}
                 <DelegateContractExpiration delegate={delegate} />
               </Box>
             )}
           </Box>
         </Box>
 
-        <Box
+        <Flex
           sx={{
-            display: 'flex',
             flexWrap: 'wrap',
-            width: bpi > 1 ? 'auto' : '100%',
-            marginTop: bpi > 1 ? 0 : 4
+            width: ['100%', '100%', 'auto', '100%', 'auto'],
+            mt: [4, 4, 0, 4, 0]
           }}
         >
           <Grid
             columns={[2]}
             sx={{
-              width: bpi > 1 ? 'auto' : '100%',
-              marginBottom: bpi > 1 ? 0 : 4
+              width: ['100%', '100%', 'auto', '100%', 'auto'],
+              mb: [4, 4, 0, 4, 0]
             }}
           >
-            <Box sx={{ mr: [4] }}>
+            <Flex sx={{ flexDirection: 'column', justifyContent: 'space-between' }}>
               <Box sx={{ mb: 3 }}>
                 <Text as="p" variant="microHeading" sx={{ fontSize: [3, 5] }}>
                   {totalStaked ? totalStaked.toBigNumber().toFormat(2) : '0.00'}
                 </Text>
-                <Text as="p" variant="secondary" color="onSecondary">
+                <Text as="p" variant="secondary" color="onSecondary" sx={{ fontSize: ['13px', '15px'] }}>
                   Total MKR delegated
                 </Text>
               </Box>
               <Box>
                 <Text as="p" variant="microHeading" sx={{ fontSize: [3, 5], color: 'secondaryMuted' }}>
                   Untracked
                 </Text>
-                <Text as="p" variant="secondary" color="onSecondary">
+                <Text as="p" variant="secondary" color="onSecondary" sx={{ fontSize: ['13px', '15px'] }}>
                   Pool participation
                 </Text>
               </Box>
-            </Box>
+            </Flex>
 
-            <Box sx={{ mr: [0, 0, 4] }}>
+            <Box sx={{ mr: [0, 0, 4, 0, 4] }}>
               <Box sx={{ mb: 3 }}>
                 <Text as="p" variant="microHeading" sx={{ fontSize: [3, 5] }}>
                   {mkrStaked ? mkrStaked.toBigNumber().toFormat(2) : '0.00'}
                 </Text>
-                <Text as="p" variant="secondary" color="onSecondary">
+                <Text as="p" variant="secondary" color="onSecondary" sx={{ fontSize: ['13px', '15px'] }}>
```

Comment:
I think we should try and stick to the scaling system from the theme, so using `fontSize: [1, 2]` here would get us 12px and 14px respectively. 

If the designs call for 13px & 15px we should discuss why. If it's absolutely necessary to target those sizes we should align the theme spec with the design (or vice versa).

Reply:
@tiago-varandas calling you here in the conversation.

Reply:
Stick to the scaling system from the theme would make sense.

Reply:
Agreed, updated back to 14 and 16px here (2, 3 on the scale)